packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -397,6 +412,36 @@ class FlutterDriverExtension {
     return EnterTextResult();
   }
 
+  Future<SetEditingStateResult> _setEditingState(Command command) async {
+    final SetEditingState setEditingStateCommand = command;
+    // TODO pass finder with command rather than using hard coded key
+    BinaryMessages.testSend(
+      SystemChannels.textInput.name,
+      SystemChannels.textInput.codec.encodeMethodCall(
+        MethodCall(
+          'TextInput.setEditingState',
+          jsonDecode(setEditingStateCommand.textEditingValue),
+        ),
+      ),
+    );
+    return SetEditingStateResult();
+  }
+
+  Future<SetMarkedTextResult> _setMarkedText(Command command) async {
+    final SetMarkedText setMarkedTextCommand = command;
+    // TODO pass finder with command rather than using hard coded key
+    BinaryMessages.testSend(
```

Comment:
This is how I was able to send messages to the engine, bypassing mocking.  The engine does receive these commands, but the textfield doesn't update in the simulator.